src/main/java/sorald/Repair.java

```diff
@@ -162,10 +167,19 @@ CtModel defaultRepair(
         CtModel model = initLauncher(launcher).getModel();
         EventHelper.fireEvent(EventType.PARSE_END, eventHandlers);
 
-        EventHelper.fireEvent(EventType.REPAIR_START, eventHandlers);
         repairModelWithInitializedProcessor(model, processor, violations);
-        EventHelper.fireEvent(EventType.REPAIR_END, eventHandlers);
+        return model;
+    }
 
+    CtModel mavenRepair(
+            Path inputDir, SoraldAbstractProcessor<?> processor, Set<RuleViolation> violations) {
+        EventHelper.fireEvent(EventType.PARSE_START, eventHandlers);
+        MavenLauncher launcher =
+                new MavenLauncher(inputDir.toString(), MavenLauncher.SOURCE_TYPE.APP_SOURCE);
```

Comment:
change to ALL_SOURCE?

Reply:
Done. I'm not quite sure why I decided to only process production code.